external/include/libcxx/math.h

```diff
@@ -1034,6 +1034,7 @@ copysign(_A1 __lcpp_x, _A2 __lcpp_y) _NOEXCEPT
     return copysign((__result_type)__lcpp_x, (__result_type)__lcpp_y);
 }
 
+#if 0
```

Comment:
Why do you comment below? Please leave the reason with comment.

Reply:
I will add reason in a separate commit, along with other suggestion fix.